lib/ui/viewer/search/search_suffix_icon_widget.dart

```diff
@@ -1,11 +1,9 @@
-import 'dart:async';
-
 import 'package:flutter/material.dart';
 import 'package:photos/ente_theme_data.dart';
 
 class SearchSuffixIcon extends StatefulWidget {
-  final Timer debounceTimer;
-  const SearchSuffixIcon(this.debounceTimer, {Key key}) : super(key: key);
+  final bool timerIsActive;
```

Comment:
This widget shouldn't know about the timer or debouncer? So it would be nicer to call this attribute `shouldShowSpinner`?